lib/docker/Dockerfile

```diff
@@ -10,11 +10,18 @@
 FROM golang:alpine AS go
 
 ARG goproxy=""
+ARG alpine_mirror="dl-cdn.alpinelinux.org"
+ARG pkg="github.com/go-rod/rod"
 
 COPY . /rod
 WORKDIR /rod
+RUN sed -i "s/dl-cdn.alpinelinux.org/$alpine_mirror/g" /etc/apk/repositories
+RUN apk update
+RUN apk add git
 RUN go env -w GO111MODULE=on && go env -w GOPROXY=$goproxy
-RUN go build ./lib/launcher/rod-launcher
+RUN git fetch --tags && \
```

Comment:
The problem of this design is that it only works for the rod-launcher in our image. We should let it work the same even if people build it themselves without our Dockerfile.

Reply:
Got it